torch_xla/csrc/device.cpp

```diff
@@ -81,12 +81,17 @@ bool ShouldUseVirtualDevice() {
   return use_virtual_device;
 }
 
-bool UseVirtualDevice() {
+bool UseVirtualDevice(bool flip_spmd_config) {
   spmd_config_is_locked = true;
   static bool use_virtual_device = ShouldUseVirtualDevice();
+  if (flip_spmd_config) {
+    use_virtual_device = !use_virtual_device;
+  }
```

Comment:
`flip_spmd_config` is a bit confusing at first glance since `UseVirtualDevice(true)` could set the value to false, is there a use case to disable the virtual device? Or could we make this `UseVirtualDevice(bool force_spmd = false)` and do

```cpp
if (force_spmd) {
   use_virtual_device = true;
}
```

Reply:
i like this idea better, thanks!

Reply:
I like this, done!